geopy/distance.py

```diff
@@ -120,6 +119,39 @@
     'GRS-67':        (6378.1600, 6356.774719, 1 / 298.25)
 }
 
+
+def lonlat(x, y, z=0):
+    """
+    Helper function that can be used for more explicit and obvious calling of the distance methods
+    >>> from geopy.distance import lonlat, distance
+    >>> newport_ri = (-71.312796, 41.49008)
+    >>> cleveland_oh = (-81.695391, 41.499498)
+    >>> print(distance(lonlat(*newport_ri), lonlat(*cleveland_oh)).miles)
+    538.3904453677203
+    :param x: longitude 
+    :param y: latitude
+    :param z: altitude
+    :return: Point(latitude, longitude, altitude)
+    """
+    return Point(y, x, z)
+
+
+def latlon(y, x, z=0):
+    """
+    Helper function that can be used for more explicit and obvious calling of the distance methods
```

Comment:
This function is very different from the previous one, so should be the description. I believe it will be confusing to the reader to see a function which transforms `(y, x, [z])` to the same `(y, x, [z])`.

Actually I'm starting to think that instead of introducing this function, it would probably be better to simply encourage the use of the Point constructor instead. What do you think?